deploy/application/php/php-apps.md

```diff
@@ -16,6 +16,28 @@ PHP is available on our platform with the branches 5.6, 7.2, 7.3, 7.4 and 8.0. Y
 
 The HTTP server is [Apache 2](https://httpd.apache.org/), and the PHP code is executed by [PHP-FPM](https://php-fpm.org/).
 
+## Memory Limit
```

Comment:
This section should be put later in the documentation, maybe after "Configure your PHP application"? Maybe it doesn't even require a `h2` title, just another sub-section.